index.html

```diff
@@ -7,93 +7,211 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+        rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700;800;900&display=swap"
+        rel="stylesheet">
     <link rel="stylesheet" href="style.css">
     <title>Mo&apos;athal&apos;s Portfolio</title>
 </head>
 
 <body>
-    <div class="container flex column">
-        <header class="toolbar flex">
-            <a class="logo" href="_blank"> Mo&apos;athal S. Kachi</a>
-            <nav class="button"></nav>
-        </header>
-
-        <section class="headline flex column">
-            <div class="head flex column">
-                <h1 class="mainTxt">Mo&apos;athal</h1>
-
-                <h3 class="spprtTxt">Software Developer</h3>
-            </div>
-
-            <div class="spprtSect flex column">
-                <p>
-                    I can help you build a product , feature or website Look through some of my work and experience! If
-                    you like what you see and have a project you need coded, don’t hesitate to contact me.
-                </p>
-
-                <hr class="divider">
-
-                <div class="sect flex">
-                    <div class="frame">
-                        
-                        <h4>Languages</h4>
-                    <ul>
-                        <li>SQL</li>
-                        <li>HTML</li>
-                        <li>CSS</li>
-                        <li>Javascript</li>
-                        <li>Ruby</li>
-                        <li>SQL</li>
-                    </ul>
-                    </div>
-
-                    <div class="frame">
-                        <h4>Frameworks</h4>
-                    <ul>
-                        <li>Flask</li>
-                        <li>Ruby on Rails</li>
-                        <li>SQlite</li>
-                        <li>Bootstrap</li>
-                    </ul>
-                    </div>
+    <div class="main-container flex column">
+        <div class="sub-container-1 flex column">
+            <header class="toolbar flex">
+                <a class="logo" href="_blank"> Mo&apos;athal S. Kachi</a>
+                <nav class="button nav"></nav>
+            </header>
+
+            <section class="headline flex column">
+                <div class="head flex column">
+                    <h1 class="mainTxt">Mo&apos;athal</h1>
+
+                    <h3 class="spprtTxt">Software Developer</h3>
+                </div>
```

Comment:
Great work @Moathal ✋

- [ ] Just like the last reviewer pointed out. Please try to align this section with grid instead of flex. This is specified in the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md).

Reply:
@1GORDON this is old before studying grid it is from the first branch